projects/demo/src/modules/components/sheet-dialog/examples/6/index.less

```diff
@@ -0,0 +1,36 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+.input {
+    inline-size: 100%;
+    margin-block-start: 0.75rem;
```

Comment:
example is cool
please add mixin `.tui-prevent-ios-scroll()` to input, to prevent shaking on ios